### ISSUE TITLE
Nearest equivalent of spring-vault-dependencies BOM

### ISSUE THREAD

**Issue by radarsh** [spring-projects/spring-vault#811]
In our project we were using spring-vault-dependencies as a BOM. However, I can see that it has been removed in the recent versions thanks to #749.

What is the closest equivalent? I tried to use spring-vault-parent but when I look at the [published POM in Maven Central](https://central.sonatype.com/artifact/org.springframework.vault/spring-vault-parent/3.0.4/overview), it seems to be missing the dependencyManagement section and therefore is not acting like a proper BOM.

**Comment by mp911de** [comment]
We used the `dependencies` artifact to define dependency versions for some (but not all) of our dependencies. We removed it to not have two places to maintain version numbers. 
Since this is a single-module project, we do not maintain a Bill of Materials as it would only list `spring-vault-core` as the sole version-managed dependency. 

We generally recommend using Spring Boot's dependency management as we closely align with driver and dependency versions there.
